core/server/api/themes.js

```diff
@@ -26,11 +27,24 @@ var debug = require('debug')('ghost:api:themes'),
 themes = {
     browse: function browse() {
         debug('browsing');
-        var result = packageUtils.filterPackages(themeList.getAll());
+        var result = themeList.toAPI(themeList.getAll(), settingsCache.get('activeTheme'));
```

Comment:
That all falls under theme-module-api refactoring which I haven't done yet 😉 

I am gettting there! Slowly, but I am!